packet/arp.go

```diff
@@ -103,23 +116,14 @@ func InitARPRequestPacket(packet *Packet, SHA [common.EtherAddrLen]uint8, SPA, T
 // TPA specify target MAC and IP addresses. Destionation MAC address
 // is L2 Ethernet header is set to THA and source address is set to
 // SHA.
-func InitARPReplyPacket(packet *Packet, SHA, THA [common.EtherAddrLen]uint8, SPA, TPA uint32) bool {
-	var bufSize uint = common.EtherLen + common.ARPLen
-	if low.AppendMbuf(packet.CMbuf, bufSize) == false {
-		common.LogWarning(common.Debug, "InitGARPAnnouncementReplyPacket: Cannot append mbuf")
+func InitARPReplyPacket(packet *Packet, SHA, THA [common.EtherAddrLen]uint8, SPA, TPA [common.IPv4AddrLen]uint8) bool {
+	if !InitARPCommonData(packet) {
```

Comment:
Please leave SPA and TPA parameter types as they were to uint32. Converting them to byte array should happen inside of service functions, not in public functions API.